Mathlib/Algebra/Polynomial/Degree/Lemmas.lean

```diff
@@ -94,6 +94,14 @@ theorem natDegree_C_mul_le (a : R) (f : R[X]) : (C a * f).natDegree ≤ f.natDeg
     _ = 0 + f.natDegree := by rw [natDegree_C a]
     _ = f.natDegree := zero_add _
 
+lemma degree_C_mul_le (r : R) (p : R[X]) : (C r * p).degree ≤ p.degree := by
+  by_cases hp : p = 0
+  · simp [hp]
+  by_cases hCp : C r * p = 0
+  · simp [hCp]
+  rw [degree_eq_natDegree hp, degree_eq_natDegree hCp]
+  simpa using natDegree_C_mul_le _ _
```

Comment:
```suggestion
lemma degree_C_mul_le (r : R) (p : R[X]) : (C r * p).degree ≤ p.degree :=
  (degree_mul_le _ _).trans <| add_le_of_nonpos_left degree_C_le
```

I believe that this might work.

Reply:
You convinced me that this lemma wasn't useful to have